packages/components/src/Filter/FilterDatePicker/FilterDatePicker.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react"
 import { formatDateAsText } from "~components/Calendar"
 import { getLocale } from "~components/DatePicker/utils/getLocale"
-import { Filter, FilterContents, FilterProps } from "../Filter"
+import { Filter, FilterContents, FilterProps } from "~components/Filter"
```

Comment:
Just checking if its intended to have these changes for the Filter in? It look like its just a bit of house keeping with the import path but wanted to confirm 😉

Reply:
Good catch